aic/__init__.py

```diff
@@ -265,6 +281,69 @@ def process_interleaved_submit(
         """Submit :py:meth:`process_interleaved` to the worker thread, returning a Future."""
         return self._executor.submit(self.process_interleaved, pcm, channels)
 
+    def process_sequential(
+        self,
+        pcm: _np.ndarray,
+        channels: int,
+    ) -> _np.ndarray:
+        """Enhance *pcm* **in-place** using sequential channel data processing.
+
+        Processes audio where all samples for each channel are stored sequentially
+        (channel 0 samples, then channel 1 samples, etc.) rather than interleaved.
+
+        Parameters
+        ----------
+        pcm
+            Sequential 1-D array of shape *(frames * channels,)* containing sequential audio data
```

Comment:
How does this sort of formatting work? Wouldn't this result in something like **(frames** * channels,)* ?